java_gen/pre-written/src/main/java/org/projectfloodlight/openflow/types/MacAddress.java

```diff
@@ -37,6 +37,8 @@
     private static final long LLDP_MAC_ADDRESS_VALUE = 0x0180c2000000L;
     private final static MacAddress MULTICAST_BASE_ADDRESS =
            MacAddress.of("01:00:5E:00:00:00");
+    private final static MacAddress IPv6_MULTICAST_BASE_ADDRESS =
+           MacAddress.of("33:33:00:00:00:00");
```

Comment:
`IPv6_*` -> `IPV6_*`

Use uppercase for the "v" to conform to Java naming conventions.